src/sources/file_descriptors/file_descriptor.rs

```diff
@@ -29,13 +30,16 @@ pub struct FileDescriptorSourceConfig {
     pub host_key: Option<OptionalValuePath>,
 
     #[configurable(derived)]
+    #[configurable(metadata(docs::hidden))]
```

Comment:
I'm actually not sure if this is necessary/correct/what to do  🤔 , @tobz  probably knows.

The `FramingConfig` is documented already in `lib/codecs/src/encoding/mod.rs` , so I'm wondering if adding this docs hidden change is actually a counter-productive thing.

Reply:
Oh I just re-read

> errors out due to duplicate entries for decoding and framing doc without that change.

hmmm 🤔 it _feels_ like it shouldn't be necessary to explicitly hide it if it's already documented where it's defined.

Reply:
You can add a `auto_generated: true` to https://github.com/vectordotdev/vector/pull/15855/files#diff-704bafbf43a586c4f12a330f7bd7503cb4bf05046437d6c4096f9e7a59d95752R15-R16, think that should resolve the problem

Reply:
Yeah, adding `auto_generated: true` is the correct solution.